templates/github-workflows/check_helm_chart_version.yaml

```
@@ -0,0 +1,52 @@
+name: Helm Chart Version Check
+on:
+  pull_request:
+
```

```suggestion
on:
  pull_request:
    branches: 
      - main
```
Perhaps the workflow should only trigger when there is a pull request to the main branch